delivery/docker/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM maven:3.6.1-jdk-11 as mvn
+ENV MAVEN_HOME /usr/share/maven
+ENV MAVEN_CONFIG "/root/.m2"
+ENV MAVEN_OPTS "-Dmaven.wagon.http.ssl.insecure=true -Dmaven.wagon.http.ssl.allowall=true"
+
+ARG AWS_SECRET_ACCESS_KEY
+ARG AWS_ACCESS_KEY_ID
+
+WORKDIR /opt/app
+
+COPY pom.xml ./pom.xml
+COPY src/ ./src/
```

```suggestion
COPY pom.xml ./pom.xml

RUN mvn dependency:resolve-plugins dependency:resolve clean package

COPY src/ ./src/
``` 
maybe this change for improving the compilation time when you don't change the pom?

---

ok, but notice that I will run this dockerfile once in life...